debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java

```diff
@@ -280,4 +293,40 @@ public void ensureConnectorTaskState(String connectorName, int taskNumber, Conne
                 .atMost(waitTimeForRecords() * 5, TimeUnit.SECONDS)
                 .until(() -> getConnectorTaskState(connectorName, taskNumber) == status);
     }
+
+    public static String getDebeziumStableVersion() {
```

Comment:
@gunnarmorling I have added the implementation for the stable debezium version here.